test/js/bun/ffi/ffi.test.js

```diff
@@ -927,14 +927,6 @@ const libSymbols = {
     returns: "int",
     args: ["ptr", "ptr", "usize"],
   },
-  pthread_attr_getguardsize: {
```

Comment:
?

Reply:
Duplicate property. It's declared above.